backend/databasemodel/model.sql

```diff
@@ -1367,19 +1370,18 @@ CREATE TABLE kooste.lipas_viivat (
 	"phoneNumber" text,
 	www text,
 	season text NOT NULL,
-	toilet boolean DEFAULT NULL,
-	"restPlacesCount" numeric DEFAULT NULL,
-	"skiTrackTraditional" boolean DEFAULT NULL,
-	"skiTrackFreestyle" boolean DEFAULT NULL,
-	"litRouteLengthKm" numeric DEFAULT NULL,
-	"routeLengthKm" numeric DEFAULT NULL,
-	"accessibilityInfo" text DEFAULT NULL,
-	"exerciseMachines" boolean DEFAULT NULL,
-	"infoFi" text DEFAULT NULL,
-	"shootingPositionsCount" numeric DEFAULT NULL,
-	status text DEFAULT NULL,
```

Comment:
Should we add `visibility` field if we remove `status`?

Reply:
This is something we discussed via Slack. We decided (with Sanna) to leave visibility field out from lipas tables since we want to show each feature in lipas belonging to the predefined type codes. If there is something they do not want to show, lipas will be used for deleting such objects.